lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -78,7 +97,7 @@ sub handle_exception {
 
 $json_schemas{version_info} = joi->object->strict;
 sub version_info {
-    my ( $self ) = @_;
+    my $self = shift;
```

Comment:
Could you change these back to `my ( $self ) = @_;` for consistency?

Reply:
done in 52852d5